impl_core/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

```diff
@@ -327,6 +345,13 @@ void combine(MutableAggregation other, double fraction) {
       for (int i = 0; i < bucketCounts.length; i++) {
         this.bucketCounts[i] += bucketCounts[i];
       }
+
+      for (int i = 0; i < mutableDistribution.getExemplars().length; i++) {
+        Exemplar exemplar = mutableDistribution.getExemplars()[i];
+        if (exemplar != null) {
+          this.exemplars[i] = exemplar;
```

Comment:
Why does this method prefer the exemplar from `mutableDistribution`?  Is it always newer?

Reply:
> Is it always newer?

Correct, this has the same logic to `LastValue` (see https://github.com/census-instrumentation/opencensus-java/pull/1055/files#diff-21149fa86227f74f9e298e6813e88275R397 and https://github.com/census-instrumentation/opencensus-java/pull/1055/files#diff-43fa0dd191ba5d49938303943d995ee0R400). I added some explanation for it.